volume/drivers/coprhd/coprhd.go

```diff
@@ -217,40 +408,82 @@ func (d *driver) Snapshot(
 	volumeID string,
 	readonly bool,
 	locator *api.VolumeLocator) (string, error) {
-	return "", nil
+	return "", volume.ErrNotSupported
 }
 
 func (v *driver) Status() [][2]string {
 	return [][2]string{}
 }
 
-// getAuthSession returns an authenticated API Session
-func (d *driver) getAuthSession() (session *napping.Session, err error) {
-	e := ApiError{}
-
-	s := napping.Session{
-		Userinfo: d.creds,
-		Client:   d.httpClient,
+func (d *driver) Inspect(volumeIDs []string) ([]*api.Volume, error) {
+	vols, err := d.DefaultEnumerator.Inspect(volumeIDs)
+	if err != nil {
+		return nil, err
 	}
 
-	url := d.url + loginUri
-
-	resp, err := s.Get(url, nil, nil, &e)
+	for i, v := range vols {
+		proto := d.itr.Protocol
```

Comment:
protocol https://github.com/libopenstorage/openstorage/blob/master/STYLEGUIDE.md (search for "Variable names")